benchmarks/tpch-gen.sh

```diff
@@ -21,14 +21,15 @@
 pushd ..
 . ./dev/build-set-env.sh
 popd
-docker build -t datafusion-tpchgen:$DATAFUSION_VERSION -f tpchgen.dockerfile .
+
+docker build  -f tpchgen.dockerfile -t datafusion-tpchgen:$DATAFUSION_VERSION .
 
 # Generate data into the ./data directory if it does not already exist
 FILE=./data/supplier.tbl
 if test -f "$FILE"; then
     echo "$FILE exists."
 else
   mkdir data 2>/dev/null
-  docker run -v `pwd`/data:/data -it --rm datafusion-tpchgen:$DATAFUSION_VERSION
```

Comment:
This change specifically breaks the volume mount for me. It appears the docker syntax expects the `imagename:version args` to come last, and putting it before the `-v` causes the `-v` to be interpreted as an argument to the script, not a volume mount. As such, the files are generated into an unbound volume and deleted when the container is destroyed.

Reply:
I created PR #2752 to resolve.